src/main/docs/guide/runningTheApp.adoc

```diff
@@ -1 +1,6 @@
 include::{commondir}/common-runapp.adoc[]
+
```

Comment:
this guide is not about basic auth. We should remove this.

Reply:
done